project-communications/applications/GSoC-2021.md

```diff
@@ -0,0 +1,83 @@
+# GSoC 2021 application summary
+
+## Google Summer of Code (GSoC) 2021
+
+-   Website: [https://summerofcode.withgoogle.com/](https://summerofcode.withgoogle.com/) and [https://google.github.io/gsocguides](https://google.github.io/gsocguides)     
+-   Context: Google Summer of Code is a global program focused on bringing more student developers into open source software development. Students work with an open source organization on a 10-week programming project during their break from school.
+- [INCF call for the organisation partners](https://neurostars.org/t/incf-call-for-community-project-proposals-and-mentors-for-gsoc-2021-deadline-for-expressions-of-interest-january-15/17865), [supporting document with details](https://docs.google.com/document/d/1T7QCv5WDxqUt1FNHiAhbE74c7e7eutzFuv3wrqToVyo/edit)
+- Submission Date: 15 January 2020
+
+### Title
+
+* AutSPACEs: a co-created citizen science platform 
+
```

Comment:
```suggestion

### Tech Keywords
GitHub, Python, Django, UI/UX design, web development
```

Reply:
Moving the tech keywords to the top to avoid breaking the flow of project description below